src/vm/excep.cpp

```diff
@@ -5665,6 +5665,8 @@ LONG __stdcall COMUnhandledExceptionFilter(     // EXCEPTION_CONTINUE_SEARCH or
         return retVal;
     }
 
+#ifndef FEATURE_CORECLR
+#ifdef FEATURE_CORRUPTING_EXCEPTIONS
```

Comment:
Hmm I thought that's what I had originally. With this feature disabled, if all of the first four conditions fail, I had an _ASSERTE(false) to fail.